### PR TITLE
Enables plugin bundling

### PR DESCRIPTION
**Pull request by distantnative**
Builds on top of https://github.com/getkirby/kirby/pull/385

This PR enables plugin modules/bundles so far for:
- [x] Fields
- [x] Widgets
- [x] Blueprints

Would appreciate :eyes: for testing :+1:

### CODE REVIEW

File: app/src/panel/models/page/blueprint/pages.php [hunk 1]
```diff
@@ -22,7 +22,7 @@ class Pages extends Obj {
   public function __construct($params = array()) {
 
     if($params === true) {
-      $this->template = blueprint::all();
+      $this->template = true;
```
Comment (lukasbestle) [inline app/src/panel/models/page/blueprint/pages.php]: `true` is a bit misleading as a general "special value". Maybe this should be a class constant with an integer value (like `self::DETECT_DYNAMICALLY`) to make clear what it means.

Comment (distantnative) [inline app/src/panel/models/page/blueprint/pages.php]: Not sure about this - also not sure if I understand you completely, could you maybe make your example more explicit? The thing is that within the blueprint logic, we always use `true` as a representation for `all`. So forwarding the `true` value from `$params` to `$this->template` and only delaying the trigger of `blueprint:all()` seemed appropriate.

Comment (lukasbestle) [inline app/src/panel/models/page/blueprint/pages.php]: Oh, do we? I didn't know we already used `true` with the meaning `all`. If so, that's completely fine. :)

Comment (distantnative) [inline app/src/panel/models/page/blueprint/pages.php]: Well at least on a broader scale `pages: true` always implies `templates: all`... Maybe @bastianallgeier knows a nicer way to still get the complete list from `blueprint:all()` without delaying it to lines 54-56.

**Pull request closed without merging**